extended_mypy_django_plugin/_plugin/annotation_resolver.py

```diff
@@ -213,7 +216,8 @@ def _concrete_for(
             found = model_type.item
 
         if isinstance(found, AnyType):
-            self.fail("Tried to use concrete annotations on a typing.Any")
+            if self._defer():
```

Comment:
I'm totes gonna follow this up later with a change that says this function returns True only if it could defer, rather than currently where it returns True if it couldn't